examples/aruba-uxi-example/values-staging.yaml

```
@@ -1,6 +1,10 @@
 aruba-uxi:
   sealedSecrets:
     imagePullSecret: sealed_version_of_the_base64_encoded_dockerconfigjson
+    secretMount:
+      firebase-service-account:
+        SERVICE_ACCOUNT.json:
```

```suggestion
        google_service_account.json:
```

Your name for it was fine, files can have any name but all caps files just look so bad 😅

---

Yes, I agree. Changing it to lowercase.